src/jupyter_contrib_nbextensions/install.py

```diff
@@ -9,7 +9,10 @@
 from jupyter_nbextensions_configurator.application import (
     EnableJupyterNbextensionsConfiguratorApp,
 )
-from notebook.notebookapp import list_running_servers
+try:
+    from notebook.notebookapp import list_running_servers
+except ModuleNotFoundError:
+    from jupyter_server.serverapp import list_runnning_servers
```

Comment:
Small typo here I think, `list_runnning_servers` should be `list_running_servers`